templates/support/index.html

```diff
@@ -139,578 +171,731 @@ <h2 class="p-heading--4">Security, support and managed open source for enterpris
+        </div>
+        <div class="col-6 col-medium-3">
+          <p>Whether you're running on a desktop, server or on IoT devices, ensure full integration, security and interoperability of your workloads with snaps. Get full stack support for all Canonical-maintained snaps published on the Snap Store.</p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-start-medium-2">
+          <hr class="p-rule" />
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-3 col-medium-2 col-start-medium-2">
+          <h3 class="p-heading--5">OCI-compliant container images</h3>
+        </div>
+        <div class="col-6 col-medium-3">
+          <p>In addition to the <a href="https://opencontainers.org/">OCI</a>-compliant container images listed in the Canonical <a href="https://github.com/orgs/canonical/packages?ecosystem=container&tab=packages&ecosystem=container&q=charmed-">GitHub</a>, <a href="http://hub.docker.com/u/ubuntu/">Docker hub</a> and <a href="https://gallery.ecr.aws/ubuntu">Amazon ECR</a>, get full support for:</p>
+          <ul class="p-list--divided">
+            <li class="p-list__item has-bullet"><a href="https://canonical-rockcraft.readthedocs-hosted.com/en/latest/explanation/rocks/">ROCKs</a>, Canonical's new generation of OCI-compliant, container images, built on top of Ubuntu LTS and optimised to run applications</li>
+            <li class="p-list__item has-bullet"><a href="https://ubuntu.com/engage/chiselled-ubuntu-images-for-containers">Chiselled Ubuntu</a>, Canonical's distroless container images that only fit the strictly required dependencies</li>
+          </ul>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-start-medium-2">
+          <hr class="p-rule" />
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-3 col-medium-2 col-start-medium-2">
+          <h3 class="p-heading--5">Charms: Juju-based software operators</h3>
+        </div>
+        <div class="col-6 col-medium-3">
+          <p>Revolutionise the speed and quality of software operations in distributed systems with <a href="https://juju.is/">Juju</a>, an open source orchestration engine for software operators that enables the deployment, integration and lifecycle management of applications at any scale, on any infrastructure.</p>
+          <p>Get full support on all Canonical maintained Charms, the operators that encapsulate business logic in reusable software packages that automate every aspect of an application's life.</p>
+        </div>
+      </div>
+    </div>
   </div>
 </section>
 
-<section class="p-strip" id="apps-support">
-  <div class="u-fixed-width">
-    <h2>Apps support</h2>
-    <p>Full-stack support for open source apps with quick response time guaranteed through subscription SLAs. Quickly resolve technical support issues with your key applications wherever they are running.</p>
-  </div>
 
```

Comment:
remove the extra line break

Reply:
There is no extra line break here, this is code that has been removed